src/sql-editor/standardSql/statementPositionResolversRegistry.ts

```diff
@@ -141,7 +141,8 @@ export function initStatementPositionResolvers(): StatementPositionResolversRegi
       resolve: (currentToken, previousKeyword, previousNonWhiteSpace, previousIsSlash) => {
         return Boolean(
           previousKeyword?.value.toLowerCase() === WHERE &&
-            (previousNonWhiteSpace?.isString() ||
+            ( previousNonWhiteSpace.is(TokenType.Operator, 'and') || previousNonWhiteSpace.is(TokenType.Operator, 'or') ||
```

Comment:
isn't that a problem that `and` and `or` are lowercase here?

Reply:
Nope, `is` function takes care of case handling.

Reply:
awesome